### PR TITLE
Make argb values with no alpha return MapColors#NONE

### PR DESCRIPTION
**Pull request by LooFifteen**
This PR allows `MapColors#closestColor` to return colour `NONE` when no alpha value is provided.

[Related Discussion](https://github.com/Minestom/Minestom/discussions/692)

### CODE REVIEW

File: src/main/java/net/minestom/server/map/MapColors.java [hunk 1]
```diff
@@ -201,6 +201,7 @@ private static void fillRGBArray() {
     }
 
     public static PreciseMapColor closestColor(int argb) {
+        if (argb << 24 == 0) return new PreciseMapColor(NONE, Multiplier.x1_00);
```
Comment (jglrxavpok) [inline src/main/java/net/minestom/server/map/MapColors.java:204]: That's not ARGB but RGBA.

File: src/test/java/map/TestMapColors.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+public class TestMapColors {
+
+    @Test
+    public void closestColor() {
+        assertEquals(MapColors.closestColor(0x539D87FF).getBaseColor(), MapColors.ICE);
+    }
+
+    @Test
+    public void closestColorNoAlpha() {
+        assertEquals(MapColors.closestColor(0xFFFFFF00).getBaseColor(), MapColors.NONE);
```
Comment (jglrxavpok) [inline src/test/java/map/TestMapColors.java:17]: Ditto

**Review by jglrxavpok — CHANGES_REQUESTED** [review]
While I agree with the intent of this PR, you risk breaking all current code using this API.
I do think alpha support is interesting, but we should not change too much from the current format (RGB -> RGBA requires a shift in addition to specifying the alpha bits, while RGB->ARGB only requires the addition of alpha bits)

File: src/main/java/net/minestom/server/map/MapColors.java (hunk 1 above)
Comment (SLLCoding) [reply src/main/java/net/minestom/server/map/MapColors.java:194]: Oh, whoops

**Comment by KrystilizeNevaDies** [comment]
Closed due to no longer being relevant. Feel free to reopen.

**Pull request closed without merging**